testing/btest/core/tcp-padding.zeek

```diff
@@ -0,0 +1,6 @@
+# Check that data contained in the UDP padding does not make it into protocol analysis
```

Comment:
Minor nit, suspect both of these are about ethernet padding.

```suggestion
# Check that data contained in the ethernet padding does not make it into protocol analysis
```

Reply:
Thanks :)